pkg/testutil/nerdtest/requirements.go

```diff
@@ -416,3 +417,15 @@ var RemapIDs = &test.Requirement{
 		return false, "snapshotter does not support ID remapping"
 	},
 }
+
+var ContainerdVersionV2 = &test.Requirement{
```

Comment:
```suggestion
func ContainerdVersion(v string) = &test.Requirement{
```

So that we do not need to add `ContainerdVersionV2_1`, `ContainerdVersionV2_2`, `ContainerdVersionV3`, ... for each of the versions

Reply:
`test.Requirement` is struct, how to set to `func ContainerdVersion(v string)`?

Reply:
I meant to make it a function that returns `*test.Requirement`

Reply:
got it, thanks.